config/locales/en.yml

```diff
@@ -44,7 +44,8 @@ en:
       claim_school: "Which school were you employed at between 6 April XXXX and 5 April XXXX?"
       employment_status: "Are you still employed to teach at a school in England?"
       current_school: "Which school are you currently employed at?"
-      mostly_teaching_eligible_subjects: "Did you teach eligible subjects for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?"
+      subjects_taught: "Did you teach any of the following subjects between 6 April XXXX and 5 April XXXX?"
+      mostly_teaching_eligible_subjects: "Did you teach %{subjects} for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?"
```

Comment:
This might be out of scope for this change, but I'm not sure this wording is clear.

> Did you teach Biology or Chemistry for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?

This could be read as:

> Did you teach (Biology or Chemistry) for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?

or:

> Did you teach Biology for more than 50% of your teaching time or Chemistry for more than 50% of your teaching time between 6 April XXXX and 5 April XXXX?

This might be clearer if we rewrote it as something like:

> Did teaching Biology and Chemistry cover 50% of your teaching time between 6 April XXXX and 5 April XXXX?

Reply:
Yeah, I like that 👍